controllers/gslb_controller.go

```diff
@@ -248,12 +249,18 @@ func (r *GslbReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			}
 		}
 
+		err = controllerutil.SetControllerReference(ingressToReuse, gslb, r.Scheme)
+		if err != nil {
+			log.Err(err).Msg("Can't set the ownership reference(Ingress owns Gslb)")
```

Comment:
Consider adding information about involved Gslb and Ingress objects using zerolog structured logging:
```suggestion
			log.Err(err).
			Str("Ingress", ingressToReuse.Name).
			Str("Gslb", gslb.Name).
			Msg("Cannot set the Ingress as the owner of the Gslb")
```